### DISCUSSION TITLE
Sort Feature Request

### DISCUSSION THREAD

**Discussion by dragow0lf** [xgi/houdoku#116] [Feature Requests]
Don't really know how this would work since the app doesn't host anything and is extension based, but something like an option to search for manga belonging to certain genre or just filtering your library by genre would be nice if possible. Aniyomi for example, although is source based as well, has a filter option for some extensions

**Comment by xgi** [comment]
Yeah, I've had this as a future goal for quite a while. I'd like to add more search options/filters for extensions rather than just a text box. It's definitely something I want to do, but it requires major changes in multiple places, so it probably won't happen anytime soon.

**Comment by dragow0lf** [comment]
It's fine, we can wait :D 
thanks!
